mkdocs_monorepo_plugin/parser.py

```diff
@@ -138,6 +137,44 @@ def read(self):
         try:
             with open(self.absNavPath, 'rb') as f:
                 self.navYaml = yaml_load(f)
+
+                # This will check if there is a `docs_dir` property on the `mkdocs.yml` file of
+                # the sub folder and scaffold the `nav` property from it
+                if self.navYaml and 'nav' not in self.navYaml:
+                    if "docs_dir" in self.navYaml:
```

Comment:
Looks like we can avoid another nested if block by chaining the second check with the one at top?
```suggestion
                if self.navYaml and 'nav' not in self.navYaml and "docs_dir" in self.navYaml:
```